File: insertionSort/insertionSort.js
let insertionSort = (array) => {
    for(let i = 1; i < array.length; i++) {
        let tempValue = array[i], j = i - 1;
        while(j>=0 && array[j] > tempValue){
            array[j+1] = array[j];
            j--;
        }
        array[j + 1] = tempValue;
    }
    return array;
}

console.log(insertionSort([2,1,4,7,2]))

File: insertionSort/insertion_sort.py
def insertion_sort(array):
    for i in range(1,len(array)):
        temp_value = array[i]
        j = i - 1
        while j >= 0 and array[j]>temp_value:
            array[j + 1] = array[j]
            j -= 1
        array[j + 1] = temp_value
    return array

list = [10,4,2,5]
print(insertion_sort(list))